modules/apps/frontend-view-state/frontend-view-state-rest-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,73 @@
+info:
+    license:
+        name: "Apache 2.0"
+        url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+    title: "Frontend View State"
+    version: v1.0
+openapi: 3.0.1
+paths:
+    "/active-view/{activeViewId}":
+        get:
+            parameters:
+                - in: path
+                  name: activeViewId
+                  required: true
+                  schema:
+                      type: string
+                - in: query
+                  name: plid
+                  schema:
+                      format: int64
+                      type: integer
+                - in: query
+                  name: portletId
+                  schema:
+                      type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                        application/xml:
+                            schema:
+                                type: object
+                    description:
+                        ""
+            tags: ["ActiveView"]
+        put:
+            parameters:
+                - in: path
+                  name: activeViewId
+                  required: true
+                  schema:
+                      type: string
+                - in: query
+                  name: plid
+                  schema:
+                      format: int64
+                      type: integer
+                - in: query
+                  name: portletId
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: string
+                    application/xml:
+                        schema:
+                            type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                type: object
```

Comment:
I guess it is not possible to model this object, but I have to ask

Reply:
It's an interesting use case that I didn't know we supported. They want to store unstructured data, so whatever the JS client sends so no-fixed fields. They send a JSONObject that gets transformed to a string. And they want to return the JSONObject.